src/cdomains/floatDomain.ml

```diff
@@ -169,7 +199,7 @@ module FloatInterval = struct
   (** evaluation of the binary operations *)
   let eval_binop eval_operation op1 op2 =
     norm @@ match (op1, op2) with 
-    | Some v1, Some v2 -> eval_operation v1 v2 
+    | Some v1, Some v2 -> (match eval_operation v1 v2 with | Some (l, h) -> Some (min l h, max l h) | _ -> None) (* TODO: remove this temporary fix *)
```

Comment:
This is a quick fix for the issue, where we forgot about some cases in which the lower and upper bound is reversed. You can decide, whether we want to merge this and afterwards replace this with a more meaningful solution or just wait for a solution and merge this afterwards